src/status_im2/contexts/chat/composer/view.cljs

```diff
@@ -128,7 +124,7 @@
 
 (defn composer
   [insets]
-  (let [window-height (rf/sub [:dimensions/window-height])
+  (let [window-height (-> (rn/get-window) (:height))
```

Comment:
redundant `->` kinda hard to read, better to do not use it for such scenarios, `(:height (rn/get-window))` is proper way